src/texts.js

```diff
@@ -31,49 +42,36 @@ export const TEXTS = {
   HELP_COMMANDS: 'HELP_COMMANDS',
 }
 
-const DATA = {
-  [LANG.SK]: {
-    [TEXTS.LAST_CALL]: 'Nezabudni si dnes objednať obed :slightly_smiling_face:',
-    [TEXTS.GREETING_NEW]:
-      'Ahoj, volám sa obedbot a všimol som si ťa na kanáli <CHANNEL> ' +
-      'ale nemal som ťa ešte v mojom zápisníčku, tak si ťa poznamenávam, ' +
-      'budem ti odteraz posielať last cally, pokiaľ v daný deň nemáš nič objednané :)',
-    [TEXTS.GREETING_SAVED]:
-      'Dobre, už som si ťa zapísal :) Môžeš si teraz objednávať cez kanál ' +
-      '<CHANNEL> tak, že napíšeš `@Obedbot [tvoja objednávka]`',
-    [TEXTS.FOOD_ARRIVED]: 'Prišli obedy z <RESTAURANT> :slightly_smiling_face:',
-    [TEXTS.FOOD_WILL_NOT_ARRIVE]: 'Dneska bohužiaľ obedy z <RESTAURANT> neprídu :disappointed:',
-    [TEXTS.YOUR_FOOD_ARRIVED]: 'Prišiel ti obed <ORDER> z <RESTAURANT> :slightly_smiling_face:',
-    [TEXTS.YOUR_FOOD_WILL_NOT_ARRIVE]: 'Dneska bohužiaľ obed z <RESTAURANT> nepríde :disappointed:',
-    [TEXTS.END_OF_ORDERS]: 'Koniec objednávok <RESTAURANT>',
-    [TEXTS.UNKNOWN_ORDER]: 'Poslal/a si neznámy príkaz\nNapíš `/obedy` pre viac informácií',
-    [TEXTS.NO_DM]:
-      'Objednávanie v súkromných kanáloch bolo vypnuté, ' +
-      'pošli prosím svoju objednávku do <CHANNEL> :slightly_smiling_face:',
-    [TEXTS.MENU_LOAD_FAILED]: 'Chyba počas načítavania menu :disappointed: Menu nájdeš tu: <MENU_LINK>',
-    [TEXTS.HELP_COMMANDS]: loadText('help', 'sk'),
-  },
-}
-
-const missingData = []
-
-Object.values(LANG).forEach(lang => {
-  Object.values(TEXTS).forEach(text => {
-    if (!DATA[lang][text]) {
-      missingData.push(`${lang}.${text}`)
-    }
-  })
-})
-
-if (missingData.length > 0) {
-  throw new Error(`Missing texts:\n${missingData.join('\n')}`)
-}
-
 export function getText(lang, name, placeholders = {}) {
-  const text = DATA[lang][name]
+  const text = TEXT_MESSAGES[lang][name]
 
   return Object.entries(placeholders).reduce(
     (acc, [key, value]) => acc.replace(new RegExp(`<${key}>`, 'g'), value),
     text,
   )
 }
```

Comment:
Would be nicer if this function was declared after the TEXT_MESSAGES, as this variable is declared after this function. Seems like it was this case before you changed/moved the code.

Reply:
moved.